new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ExecCommand.java

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
did you copy this from somewhere else? the copyright dates look wrong

Reply:
yes. it is copied from the existing integration test code.

Reply:
I did not change it since the class does exist since 2018. Shall I change it?

Reply:
so you are saying you copied it from somewhere else?

Reply:
yes, I copied both ExecCommand and ExecResult from integration-tests/src/test/java/oracle/kubernetes/operator/utils dir.

Reply:
ok fair enough